docssrc/src/commandarguments.md

```diff
@@ -0,0 +1,202 @@
+# CommandArguments
+
+The `CommandArguments` class was introduced in CommandAPI 9.0.0 and provides a much more powerful way of accessing arguments than just an array of arguments which existed until 9.0.0.
+
+While the argument array just gives the possibility to access the arguments via the array notation (`args[0]`), the `CommandArguments` class offers much more, including:
+
+- [Access the inner structure directly](#access-the-inner-structure-directly)
+- [What terms are used?](#what-terms-are-used)
+  - [`nodeName`](#nodename)
+  - [`raw argument`](#raw-argument)
+  - [`unsafe argument`](#unsafe-argument)
+- [Access arguments](#access-arguments)
+  - [Access arguments by node name](#access-arguments-by-node-name)
+  - [Access arguments by index](#access-arguments-by-index)
+- [Access raw arguments](#access-raw-arguments)
+  - [Access raw arguments by node name](#access-raw-arguments-by-node-name)
+  - [Access raw arguments by index](#access-raw-arguments-by-index)
+- [Access unsafe arguments](#access-unsafe-arguments)
+  - [Access arguments by node name](#access-arguments-by-node-name-1)
+  - [Access arguments by index](#access-arguments-by-index-1)
+
+ATTENTION!!!!! Every executable and generated documentation has to be deleted before committing anything!
+
+-----
+
+## Access the inner structure directly
+
+To provide arguments, the `CommandArguments` class stores:
+
+- a `Object[]` of parsed arguments
+- a `Map<String, Object>` of parsed arguments mapped to their node names
+- a `String[]` of raw arguments
+- a `Map<String, String>` of raw arguments mapped to their node names
+- a `String` which holds the full input
```

Comment:
I don't really like how this is laid out. I think it may be better to give a full explanation in a block rather than a few words which could lead to ambiguity or otherwise. An example of a suitable layout may be this, utilizing subheadings to collate similar methods (such as a section for parsed arguments, a section for raw arguments, a section for other utility methods): https://commandapi.jorel.dev/9.0.3/commandregistration.html#commandapicommand-methods